Heck/Animation/Events/CoroutineEventManager.cs

```diff
@@ -41,6 +43,24 @@ internal void StartEventCoroutine(CustomEventData customEventData, EventType eve
                 return;
             }
 
+            IEnumerable<Trigger> triggers = heckData.Triggers;
```

Comment:
Nitpick but use `triggers.Any(x => x.isTriggered)` for more declarative syntax here

Reply:
done, thanks!